lisp_ease/src/queries/grasping.lisp

```diff
@@ -0,0 +1,298 @@
+(in-package :le)
+
+;; orientation source: cram_knowrob_pick_place
+
+;; grasping offsets -------------------------------------------------------------
+
+(defparameter *lift-z-offset* 0.15 "in meters") ;0.15
```

Comment:
Get rid of all defparameters, as they are not used in lisp_ease.
Original place is cram_knowrob/cram_knowrob_pick_place/src/grasping.lisp